articles/introduction-to-r/index.md

```diff
@@ -0,0 +1,315 @@
+# Introduction to R Programming
+
+The data science community has been growing over the past decade. The tools used in the industry have been powering this unprecedented growth. Hence, learning a programming language like R empowers us to contribute to the field. This article will clarify all the basic concepts of R and look at the various concepts. Going through the article should give you sufficient information to start coding in R. 
```

Comment:
the tools used in industry => remove "the"

"This article will clarify all the basic concepts of R and look at the various concepts" => redundant, say what concepts will actually be covered: vectors, matrices, and elements

"Going through the article should give you sufficient information to start coding in R. " => more specific would be Going through this tutorial will familiarize you with the building blocks of an R program.